Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Hud small fixes

## Changelog:
- Sidebar background expanded to fit Unions tab:
![image](https://user-images.githubusercontent.com/35965798/154862807-97db2b9b-a843-4758-a809-877964b61544.png)

- [#864] Theologian Trait now inclues holy light icon, a totem and elune moon:
![image](https://user-images.githubusercontent.com/35965798/154862883-2d0af98a-7dce-49f1-aaf1-2dc27c6275d4.png)

Amplified (Looks awful lol):
![image](https://user-images.githubusercontent.com/35965798/154863043-51c152f7-8348-478c-8e17-82996eb39183.png)


## How to test:
- Check Trait icon in both lifestyle tree and Char information.
- Check if the sidebar looks good
